### PR TITLE
CB-10034 Addresses Embedded/Linked/Signed Custom Frameworks via updated node-xcode

### PR DESCRIPTION
**Pull request by robertoandrade**
- Moved previously committed node-xcode extension to that project. Waiting PR: https://github.com/alunny/node-xcode/pull/79 and https://github.com/alunny/node-xcode/pull/88
- Added options and tests for iOS frameworks to not be optionally "embedded" or "linked" depending on the use cases.

### CODE REVIEW

File: cordova-lib/package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
         "underscore": "1.7.0",
         "unorm": "1.3.3",
         "valid-identifier": "0.0.1",
-        "xcode": "0.8.0"
+        "xcode": "0.8.6"
```
Comment (robertoandrade) [inline cordova-lib/package.json]: Will confirm this is indeed the released version as soon as alunny/node-xcode#88 gets merged as a release is pushed out.

Comment (robertoandrade) [inline cordova-lib/package.json]: Would you guys prefer I point that by git fork instead just to get the build/tests passing? I'd think pointing at a non-released version would also make it so this is not mergeable, so not sure what the best state is to leave it at.

**Comment by stevengill** [comment]
Thoughts on this @shazron? Something still needed?

**Comment by shazron** [comment]
@stevengill yes. it adds the attribute, which can't be in the Platform API and must be in cordova-lib
The PR needs a rebase and/or resolution of conflicts - pinging @robertoandrade

**Comment by shazron** [comment]
@robertoandrade I want to get this in, any word? If not I can just fork your branch and do the edits needed

**Comment by robertoandrade** [comment]
Will check it out let you guys know.

**Comment by shazron** [comment]
I need to get the `embed` attribute feature out and thus will send out a new PR by just enabling:
```
embed: isStrTrue(el.attrib.embed),
```
in `cordova-common/src/PluginInfo/PluginInfo.js`.

The rest of the code will be in the Platform API of cordova-ios. See my last comment on https://issues.apache.org/jira/browse/CB-11233

**Comment by raphinesse** [comment]
The files modified for this PR have been removed in 64fea70ea0554458ebc399f0ad4ba4b534b5cd3f. Since plugin handlers now live in the platform repositories, this has to be implemented in [apache/cordova-ios](https://github.com/apache/cordova-ios).

@shazron Maybe you want to open an issue in cordova-ios that links to this, so it's not forgotten?

**Comment by shazron** [comment]
Not sure what you mean @raphinesse . All the code is in the platform already, the only part that cordova-ios depends on is PluginInfo.js which is in cordova-common which needs to parse the `embed` attribute: https://github.com/apache/cordova-common/blob/8bb30fc49996ddab22366b40185b2a828f4664f7/src/PluginInfo/PluginInfo.js#L336

**Comment by raphinesse** [comment]
@shazron I don't have any idea of what this PR once tried to do or if it has since been achieved. I closed it since all major modifications done in this PR were on files that have since been deleted or moved elsewhere. I pinged you since were the last member to comment on this PR and so hopefully would know whether there was still work to be done.

So if I understand you correctly, the underlying issue of this PR has already been solved?

**Comment by shazron** [comment]
Me neither - it's just too old for me to recall. All I know is I implemented the `embed` feature of this PR in another PR referenced above.

**Comment by raphinesse** [comment]
Well with no one knowing what to do or if there's anything to do at all, I guess this PR is rightly closed.

If this is still an issue for anyone, please don't hesitate to comment or raise an issue.

**Pull request closed without merging**